CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKRenderable.java

```diff
@@ -64,11 +64,17 @@ public CVKRenderable(CVKVisualProcessor visualProcessor) {
     
     protected CVKGraphLogger GetLogger() { return cvkVisualProcessor.GetLogger(); }
     
-    /*
-        Returns the command buffer for the current Image being sent
-        to the GFX drivers
-    */
-    public abstract VkCommandBuffer GetCommandBuffer(int imageIndex);        
+    /**
+     * Returns the command buffer for the current Image being sent
+     * to the GFX drivers
+     */
+    public abstract VkCommandBuffer GetDisplayCommandBuffer(int imageIndex);        
```

Comment:
As mentioned before, every renderable should name its commandbuffers according to display or hittest.

Reply:
Renamed